hs_collection_resource/tests/test_collection.py

```diff
@@ -366,25 +366,6 @@ def test_update_collection_edit_permission(self):
         self.assertEqual(self.resCollection.resources.count(), 1)
         self.assertIn(self.resGen3, self.resCollection.resources.all())
 
-    def test_adding_one_collection_to_another_collection(self):
```

Comment:
Yeah, this test function appears to test a collection cannot be added to another collection as currently being implemented (i.e., it results an error). So this test function is in sync with the current implementation. When the collection nesting is implemented, you can modify this test function to test adding one collection into another collection would succeed.